src/docfx/lib/markdown/MarkdownUtility.cs

```diff
@@ -75,20 +88,31 @@ internal static void LogError(Error error)
             t_status.Value.Peek().Errors.Add(error);
         }
 
-        internal static string GetLink(string path, object relativeTo, object resultRelativeTo, MarkdownObject origin, int columnOffset = 0)
+        internal static string GetLink(string path, object relativeTo, MarkdownObject origin, int columnOffset = 0)
         {
             var status = t_status.Value.Peek();
-            var (error, link, _) = status.Context.DependencyResolver.ResolveLink(new SourceInfo<string>(path, origin.ToSourceInfo(columnOffset: columnOffset)), (Document)relativeTo, (Document)resultRelativeTo);
+            var (error, link, linkType, _) = status.Context.DependencyResolver.ResolveLink(new SourceInfo<string>(path, origin.ToSourceInfo(columnOffset: columnOffset)), (Document)relativeTo);
```

Comment:
Can we use the same way as ResolveXref to determine whether this is a relative path?
```
if (doc != null)
{
    link = RelativeUrlMarker + link
}
``` 

So then we don't need change the return value

Reply:
Just tried, there are some subtle differences: when the link target does not exist, `doc` is null, but the links should still be treated as relative link.

Reply:
When the line is not resolved, we leave it there, should be fine right?